samps/uiname.properties

```diff
@@ -1,10 +1,28 @@
+# Copyright (C) 1989-2022 PC2 Development Team: John Clevenger, Douglas Lane, Samir Ashoo, and Troy Boudreau.
+#
 # PC^2 GUI Plugins 
-# Sun Mar 16 06:15:49 PST 2008
+# Sat Apr 30 18:30:12 PDT 2022
+#
+# line form:  pc2_login=view_class_name
+# ex. team5=edu.csus.ecs.pc2.ui.team.TeamLegacyView
+#
+# To use the Legacy views or override the View/UI class per login:
+# 1 edit samps/uiname.properties
+# 2 uncomment line with login name, ex. administrator1 line below
+#    (the other line may remain, the UI is per login not client type)
+# 3 copy updated uiname.properties to the same directory where pc2v9.ini is installed
+#
 judge=edu.csus.ecs.pc2.ui.judge.JudgeView
+# judge1=edu.csus.ecs.pc2.ui.judge.JudgeLegacyView
 team=edu.csus.ecs.pc2.ui.team.TeamView
+# team1=edu.csus.ecs.pc2.ui.team.TeamLegacyView
 scoreboard=edu.csus.ecs.pc2.ui.board.ScoreboardView
+# scoreboard1=edu.csus.ecs.pc2.ui.board.ScoreboardLegacyView
 spectator=edu.csus.ecs.pc2.ui.judge.JudgeView
-administrator=edu.csus.ecs.pc2.ui.admin.AdministratorView
+# administrator=edu.csus.ecs.pc2.ui.admin.AdministratorView
+# override adminstrator1 UI View class
+# administrator1=edu.csus.ecs.pc2.ui.admin.AdministratorLegacyView
+# -- 

```

Comment:
Four lines above this line, the original line specifying the definition of the class to be used by "administrator" is removed, and it is replace by the next four lines.  However, all of the next four lines are comments.  This appears to effectively remove *_any_* definition of what class should be loaded for "administrator".  Was that the intent?

Reply:
in the original names, client types have no effect.
This was to remind me/us that the administrator line is not used and the default
is specified in the load class.